src/pudl_rmi/__init__.py

```diff
@@ -1,25 +1,31 @@
 """Connecting depreciation data, FERC1 and EIA for RMI."""
 
-import pkg_resources
+import os
 from pathlib import Path
 
-import pudl_rmi.make_plant_parts_eia
-import pudl_rmi.deprish
-import pudl_rmi.coordinate
-import pudl_rmi.connect_ferc1_to_eia
-import pudl_rmi.connect_deprish_to_ferc1
+import pkg_resources
+
 import pudl_rmi.connect_deprish_to_eia
+import pudl_rmi.connect_deprish_to_ferc1
+import pudl_rmi.connect_ferc1_to_eia
+import pudl_rmi.coordinate
+import pudl_rmi.deprish
+import pudl_rmi.make_plant_parts_eia  # noqa: F401
 
 REPO_DIR = Path(__file__).resolve().parent.parent.parent
-INPUTS_DIR = REPO_DIR / 'inputs'
+INPUTS_DIR = (
+    Path(os.environ.get("PUDL_RMI_INPUTS_DIR", REPO_DIR / 'inputs'))
```

Comment:
is this grabbing an environment variable if it is set but using the standard path if it is not??? is this set somewhere in the tox setup?

Reply:
Yes that's right. The environment variables are set at the top of `tox.ini` in the `setenv` section. However, they could be set anywhere in the the process, so long as they are passed through to the environment that `tox` is running in.